flexmeasures/utils/plugin_utils.py

```diff
@@ -24,12 +24,8 @@ def register_plugins(app: Flask):
     (last part of the path).
     """
     plugins = app.config.get("FLEXMEASURES_PLUGINS", [])
-    if not plugins:
-        # this is deprecated behaviour which we should remove in version 1.0
-        app.logger.debug(
-            "No plugins configured. Attempting deprecated setting FLEXMEASURES_PLUGIN_PATHS ..."
-        )
-        plugins = app.config.get("FLEXMEASURES_PLUGIN_PATHS", [])
+    if isinstance(plugins, str):
+        plugins = [plugin.strip() for plugin in plugins.split(",")]
```

Comment:
It would be nice to filter out plugin names that have length 0 (`len(plugin.strip())>0`) to avoid trailing commas, e.g `plugin1,plugin2,` -> ["plugin1", "plugin2", ""]. Maybe this is already handle elsewhere.